src/student/student.service.ts

```diff
@@ -51,6 +55,10 @@ export class StudentService {
         courseEngagement: true,
         projectDegree: true,
         teamProjectDegree: true,
+        hr: {
+          email: true,
+        },
+        status: true,
       },
```

Comment:
I can't see 'reservedTo'

Reply:
reservedTo has been added to findAll students method